src/tests/packages/excalidraw.test.tsx

```diff
@@ -353,4 +328,64 @@ describe("<Excalidraw/>", () => {
       ).toBe(true);
     });
   });
+
+  describe("<MainMenu/>", () => {
+    it("should render main menu with host menu items if passed from host", async () => {
+      const { container } = await render(
+        <Excalidraw UIOptions={undefined}>
```

Comment:
```suggestion
        <Excalidraw>
```
since its `undefined` so doesn't need to be passed

Reply:
I think I've copied from elsewhere and forgot to update

Reply:
oh, it's your test actually :p